dnsimple/domains_email_forwards.go

```diff
@@ -7,12 +7,17 @@ import (
 
 // EmailForward represents an email forward in DNSimple.
 type EmailForward struct {
-	ID        int64  `json:"id,omitempty"`
-	DomainID  int64  `json:"domain_id,omitempty"`
+	ID       int64 `json:"id,omitempty"`
+	DomainID int64 `json:"domain_id,omitempty"`
+	// For requests, this field is deprecated, please use `AliasName` instead.
 	From      string `json:"from,omitempty"`
```

Comment:
My understanding was that we would also deprecate `From` in favor of `AliasEmail`, since `AliasName` is a write-only attribute it can be misleading.

Reply:
`AliasEmail` is only used in responses. Are you suggesting we also add an `AliasEmail` field and add a comment to only use it in responses?